contactform.php

```diff
@@ -176,14 +375,17 @@ public function sendMessage()
             $this->context->controller->errors[] = $this->trans('An error occurred during the file-upload process.', array(), 'Modules.Contactform.Shop');
         } elseif (!empty($file_attachment['name']) && !in_array(Tools::strtolower(substr($file_attachment['name'], -4)), $extension) && !in_array(Tools::strtolower(substr($file_attachment['name'], -5)), $extension)) {
             $this->context->controller->errors[] = $this->trans('Bad file extension', array(), 'Modules.Contactform.Shop');
+        } elseif ($url !== '' || empty($serverToken)
+            || $clientToken !== $serverToken || $clientTokenTTL < time()
```

Comment:
Either one line for all statements or one statement per line please

Reply:
done